src/Sylius/Bundle/ApiBundle/Assigner/CartToUserAssigner.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Assigner;
+
+use Doctrine\Persistence\ObjectManager;
+use Sylius\Component\Core\Repository\OrderRepositoryInterface;
+use Sylius\Component\Customer\Model\CustomerInterface;
+use Sylius\Component\Core\Model\OrderInterface;
+use Symfony\Component\HttpFoundation\Session\SessionInterface;
+
+/** @experimental */
+class CartToUserAssigner implements CartToUserAssignerInterface
```

Comment:
There is a `CartBlamerListener` that should work instead of this assigner I guess